tools/grid/README.md

```diff
@@ -61,15 +61,6 @@ const renderItem = (
 };
 
 // ...
-
-<sp-grid
```

Comment:
Does this code break the build in some way? I don't want to block the PR on this, but if we can't return this segment of code, can you open an issue for updating it afterwords?

Reply:
Yeah, it does break the build. I'll make an issue for it.